dotcom-rendering/src/web/lib/useSDC.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ModuleDataResponse } from '@guardian/support-dotcom-components';
+import {
+	getBanner,
+	getEpic,
+	getLiveblogEpic,
+	getPuzzlesBanner,
+} from '@guardian/support-dotcom-components';
+import type {
+	BannerPayload,
+	EpicPayload,
+} from '@guardian/support-dotcom-components/dist/dotcom/src/types';
+import type { SWRConfiguration } from 'swr';
+import useSWR from 'swr';
+
+const options: SWRConfiguration = {
```

Comment:
I think instead of specifying these options you could use `useSWRImmutable` which will give the same behaviour:  https://swr.vercel.app/docs/revalidation#disable-automatic-revalidations

Reply:
nice 👍